### ISSUE TITLE
Input tarball AM4_tun.tar.gz on ftp site is broken

### ISSUE THREAD

**Issue by russfiedler** [NOAA-GFDL/AM4#18]
The input tarball has been truncated to about 2GiB and is missing quite a few files. It's possible to download the individual files from the AM4_run directory but it would be nicer to have the compress file available.

 [ftp://nomads.gfdl.noaa.gov/users/Ming.Zhao/AM4Documentation/GFDL-AM4.0/inputData](url)

**Comment by thomas-robinson** [comment]
@russfiedler Here is the link for the input tarball ftp://nomads.gfdl.noaa.gov/users/Ming.Zhao/AM4Documentation/GFDL-AM4.0/inputData/AM4_run.tar.gz  I downloaded it this morning on a new system, and it is 21GB.  Here are the results of `ls`
```bash
ls -sh AM4_run.tar.gz
21G AM4_run.tar.gz
```
It even says the size of the file is 20.7 GB on the ftp site:
```
AM4_run.tar.gz      20.7 GB	7/9/18, 8:00:00 PM
```
You may want to check that you have enough room for the file on your system, and try to redownload the file.  It seems like your current copy of the file is somehow incomplete.  Let me know if you are still having problems after you confirm there is enough space on your system and you redownload the file.

**Comment by russfiedler** [comment]
@thomas-robinson  That's odd. It was a couple of months ago that I actually tried it from 2 systems and both hit the 2 GiB limit on the file size. My first thought was that it was a problem at my end but when things failed from the NCI supercomputer facility I believed it was a problem with the file especially when I was able to download the CM4 tarball with no problems from there. Shrug, who knows what was going on. I'll close this.
